extensions/label/README.md

```diff
@@ -0,0 +1,163 @@
+| label:classes     | Class Object         | Classes                    | **REQUIRED** if using categorical data. A Class Object defining the list of possible class names for each `label:property`. (e.g., tree, building, car, hippo)|
+| label:description | string               | Description                | **REQUIRED** A description of the label, how it was created, and what it is recommended for |
+| label:type        | string               | Type                       | **REQUIRED** An ENUM of either `vector` label type or `raster` label type |
+| label:task        | [string]             | Task                       | Recommended to be a subset of 'regression', 'classification', 'detection', or 'segmentation', but may be an arbitrary value |
+| label:method      | [string]             | Method                     | Recommended to be a subset of 'automated' or 'manual', but may be an arbitrary value. |
+| label:overview    | Label Overview Object| Overview                   | An Object storing counts (for classification-type data) or summary statistics (for continuous numerical/regression data). |
+
+#### Class Object
+| Field Name      | Type                 | name                       | description       |
+|-----------------|----------------------|----------------------------|--------------------------------------------------------------------------------------------------|
+| name            | string or null       | Name                       | The property key within the asset's each `Feature` corresponding to class labels. If labels are raster-formatted, use null.|
+| classes         | [string or numeric]  | Classes                    | The different possible class values within the property `name`. |
+
+#### Label Overview Object
+
+| Field Name      | Type            | name                       | description       |
+|-----------------|-----------------|----------------------------|--------------------------------------------------------------------------------------------------|
+| property_key    | string          | Property Key                       | The property key within the asset corresponding to class labels. |
+| counts          | [Count Object]  | Counts                     | An object containing counts for categorical data. |
+| statistics      | [Stats Object]  | Statistics                 | An object containing statistics for regression/continuous numeric value data. |
+
+`label:overview ` generally won't have both counts and statistics, but one of the two.
+
+#### Count Object
+
+| Field Name      | Type            | name                       | description       |
+|-----------------|-----------------|----------------------------|--------------------------------------------------------------------------------------------------|
+| class_name      | string          | Class Name                    | The different possible classes within the property `name`. |
```

Comment:
Shorten the name from `class_name` to `name` or `id`? (see #499)

Reply:
Same with above, I think `name` would probably make more sense, but no problem changing it from `class_name`.

I'll make the updates.